src/jasper/helpers/forms.cr

```diff
@@ -36,10 +36,22 @@ module Jasper::Helpers::Forms
     content(name: :label, content: content, options: {:for => name, :id => "#{name}_label"})
   end
 
+  def csrf_field
+    hidden_field(name: "authenticity_token", value: "")
```

Comment:
I think now that we have an application_controller. We can finish this part up. I'll try to come up with an example.